### PR TITLE
Add secrets docs for local runs.

### PR DESCRIPTION
**Pull request by TSienki**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -550,6 +550,8 @@ print(config2.two) # => 102
 To define a secret for your BigFlow workflow, you need to use [Kubernetes Secrets](https://cloud.google.com/composer/docs/how-to/using/using-kubernetes-pod-operator#secret-config)
 mechanism (Cloud Composer is a GKE cluster under the hood).
 
+**Secrets management based on Kubernetes Secrets works only if a job is scheduled in Composer. When you want to run the job from a local machine secrets should be provided as environment variables.**
```
Comment (grzegorzwitkowski) [inline docs/configuration.md:553]: Is it specified somewhere else what are these env variables?

Comment (TSienki) [reply docs/configuration.md:553]: Environment variables on your computer ;)

Comment (TSienki) [reply docs/configuration.md]: I added that

Comment (grzegorzwitkowski) [reply docs/configuration.md]: I know, but what variables (names) do I have to set?

**Review by grzegorzwitkowski — APPROVED** [review]

**Pull request merged**